Storage/Storage/SQL/VisitsTable.swift

```diff
@@ -0,0 +1,53 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+* License, v. 2.0. If a copy of the MPL was not distributed with this
+* file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import Foundation
+
+let TableNameVisits = "visits"
+
+class VisitsTable<T>: GenericTable<Visit> {
+    override var name: String { return TableNameVisits }
+    override var rows: String { return "siteGuid TEXT NOT NULL, " +
```

Comment:
I left the comment on the last PR, so here it goes again: site numeric ID, visit type.